packages/modules/desktop-viewer-react/src/types.ts

```diff
@@ -22,7 +22,7 @@ export type DesktopViewerProps = XOR<
   XOR<FileViewerProps, BlankViewerProps>,
   ConnectedViewerProps
 > &
-  DesktopInitializerParams;
+  Omit<DesktopInitializerParams, "clientId">;
```

Comment:
Continuing off the last comment, not sure why we'd want to omit the clientId as a prop. Users will still need to provide this, when necessary.